phpunit/class-wp-rest-template-controller-test.php

```diff
@@ -103,6 +129,32 @@ public function test_get_item() {
 			),
 			$data
 		);
+
+		// Test template parts.
+		$request  = new WP_REST_Request( 'GET', '/wp/v2/template-parts/tt1-blocks//header' );
```

Comment:
Not a blocker, but could you point me to where `theme-name//part-slug` was decided as an identifier?

Reply:
Sure thing!  I believe that was introduced in Riad's previous PR to remove auto-drafts https://github.com/WordPress/gutenberg/pull/27910 .

Im having trouble finding the exact conversation there, but I think it was originally `theme|slug` and @Copons had made an argument to update it to `theme//slug` IIRC.

Reply:
https://github.com/WordPress/gutenberg/pull/27910#discussion_r553288550